vumi_http_retry/retries.py

```diff
@@ -20,3 +26,26 @@ def add_request(redis, prefix, req):
     }
 
     return redis.zadd(requests_key(prefix), next_score(req), json.dumps(req))
+
+
+@inlineCallbacks
+def pop_requests(redis, prefix, from_time, to_time):
+    k = requests_key(prefix)
+    reqs = yield redis.zrangebyscore(k, from_time, to_time)
```

Comment:
Sounds good. I'll make a ticket and we can add it in a new PR if txredis supports it, and otherwise leave it open until txredis does support it. Sound okay?